zebra-state/src/service/finalized_state/disk_db.rs

```diff
@@ -138,6 +146,17 @@ impl WriteDisk for DiskWriteBatch {
         self.batch.put_cf(cf, key_bytes, value_bytes);
     }
 
+    fn zs_merge<C, K, V>(&mut self, cf: &C, key: K, value: V)
+    where
+        C: rocksdb::AsColumnFamilyRef,
+        K: IntoDisk + Debug,
+        V: IntoDisk,
+    {
+        let key_bytes = key.as_bytes();
+        let value_bytes = value.as_bytes();
+        self.batch.merge_cf(cf, key_bytes, value_bytes);
```

Comment:
This PR may need to set a merge operator for this to work.

Reply:
Removed in https://github.com/ZcashFoundation/zebra/pull/7334/commits/5dd93ec4f0cd3eb3da9bac87b5bf751e72cfc780.